### ISSUE TITLE
Prod QA testing of Inverse checkbox and Carbon Theme option for several components

### ISSUE THREAD

**Issue by RobertaJHahn** [carbon-design-system/ibm-dotcom-library#3264]
#### User Story

> As a `[user role below]`:
developer using the ibm.com Library Carbon Theming in the ImagewithCaption, VideoPlayer and ContentBlockSimple components

> I need to:
have a version of the component that has been tested on multiple browsers and platforms 

> so that I can:
be confident that my ibm.com web site users will have a good experience

#### Additional information

- [Browser Stack link](https://ibm.ent.box.com/notes/578734426612)
- [Browser Standard](https://w3.ibm.com/standards/web/browser/) 
- Browser versions to be tested:  Tier 1 browsers will be tested with defects created as Sev 1 or Sev 2.  Tier 2 browser defects will be created as Sev 3 defects.
- Platforms to be tested, by priority:  1)  Desktop  2)  Mobile  3)  Tablet
- Mobile & Tablet iOS versions: 13.1 and 13.3
- Mobile & Tablet Android versions: 9.0 Pie and 8.1 Oreo 
- Browsers to be tested:  Desktop: Chrome, Firefox, Safari, Edge, Mobile: Chrome, Safari, Samsung Internet, UC Browser, Tablet: Safari, Chrome, Android
 - [Accessibility Checklist](https://www.ibm.com/able/guidelines/ci162/accessibility_checklist.html)
 - [Creating a QA bug](https://ibm.ent.box.com/notes/603242247385)
- The components included in the testing scope are ImagewithCaption, VideoPlayer and ContentBlockSimple.  These components had both the inverse theme check box and the Carbon Theme tab available, causing confusion.   This feature is removed from those components.   
- Accessibility testing is not needed for this change.
- **See the Epic for the Design and Functional specs information**
 - Dev issue (#3192) 
- ** Once development is finished the updated code is available on the [Canary URL](https://ibmdotcom-react-canary.mybluemix.net/?path=/story/overview-getting-started--page) for testing.**

#### Acceptance criteria
- [ ] All browser versions are tested
- [ ] All operating systems are tested
- [ ] All devices are tested
- [ ] Defects are recorded and retested when fixed

**Comment by RobertaJHahn** [comment]
Prod QA testing started.
